angular/src/app/cockpit-area/prediction-cockpit/prediction-cockpit.component.ts

```diff
@@ -1,14 +1,10 @@
-import { PredictionService } from '../shared/prediction.service';
-import { OrdersData } from '../../shared/view-models/interfaces';
-import { Observable } from 'rxjs/Observable';
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { FilterOrdersCockpit, PredictionCriteria } from 'app/shared/backend-models/interfaces';
```

Comment:
Here we should change the route with the relative path `../../shared/backend-models/interfaces` instead.